docs/client/full-api/tableOfContents.js

```diff
@@ -311,12 +312,14 @@ var toc = [
     "appcache",
     "accounts-ui",
     "audit-argument-checks",
+    {name: "blaze", link: "https://atmospherejs.com/meteor/blaze"},
```

Comment:
Let's get rid of Blaze and Spacebars in this list. This part of the docs so far has been a dumping ground for things we couldn't put elsewhere, and we want to reduce its size rather than increase it if possible.